requirements.txt

```diff
@@ -10,7 +10,7 @@ sh
 sqlalchemy
 pymysql
 transformers
-sentence-transformers==1.2.0
+sentence-transformers
```

Comment:
why not set the version? are you using a different version of sentence-transformers in production?

Reply:
I probably should set to 2.0.0, I had an issue with 1.2.0 downloading the model as they've moved the URL of model files since then and in a batch/empty environment it doesn't work